app/lib/actions/katello/host/hypervisors_update.rb

```diff
@@ -33,11 +33,10 @@ def run
             end
           end
 
-          @duplicate_uuid_hypervisors.each do |hypervisor, existing_host|
-            fail _("Host creation was skipped for %s because it shares a BIOS UUID with %s. " \
-                   "To report this hypervisor, override its dmi.system.uuid fact or set 'candlepin.use_system_uuid_for_matching' " \
-                   "to 'true' in the Candlepin configuration." % [hypervisor[:name], existing_host.name])
-          end
+          hypervisor, existing_host = @duplicate_uuid_hypervisors.first
+          fail _("Host creation was skipped for %s because it shares a BIOS UUID with %s. " \
+                 "To report this hypervisor, override its dmi.system.uuid fact or set 'candlepin.use_system_uuid_for_matching' " \
+                 "to 'true' in the Candlepin configuration." % [hypervisor[:name], existing_host.name])
```

Comment:
More investigation is needed but I think the changes here somehow caused `hypervisor` to become `nil`.

Reply:
@ianballou i've made changes to handle this.